src/OpenLoco/src/Paint/PaintWall.cpp

```diff
@@ -0,0 +1,130 @@
+#include "PaintWall.h"
+#include "Graphics/Colour.h"
+#include "Map/WallElement.h"
+#include "Objects/ObjectManager.h"
+#include "Objects/WallObject.h"
+#include "Paint.h"
+#include "ScenarioManager.h"
+#include "Ui.h"
+#include <cassert>
+
+using namespace OpenLoco::Interop;
+using namespace OpenLoco::Ui::ViewportInteraction;
+
+namespace OpenLoco::Paint
+{
+    static constexpr World::Pos3 kOffsets[4] = {
+        { 0, 0, 0 },
+        { 1, 31, 0 },
+        { 31, 0, 0 },
+        { 2, 1, 0 },
+    };
+
+    static constexpr World::Pos3 kBBoxOffsets[4] = {
+        { 1, 1, 0 },
+        { 2, 30, 0 },
+        { 30, 2, 0 },
+        { 1, 1, 0 },
+    };
+
+    static constexpr World::Pos3 kBBoxLengths[4] = {
+        { 1, 28, 0 },
+        { 29, 1, 0 },
+        { 1, 29, 0 },
+        { 28, 1, 0 },
+    };
+
+    static constexpr uint8_t kImageOffsets[4][3] = {
```

Comment:
I have been trying to document the image offsets as we implement the paint functions by defining ImageIds in the object files. Could you do this for the wall. (See TrainStationObject for example or ScaffoldingObject